src/gensxg.c

```diff
@@ -390,7 +390,11 @@ static void load_content(const char* filepath, sxg_buffer_t* buf) {
   }
   rewind(file);
   sxg_buffer_resize(filesize, buf);
-  fread(buf->data, sizeof(uint8_t), filesize, file);
+  int64_t nread = fread(buf->data, sizeof(uint8_t), filesize, file);
```

Comment:
Given the duplication of fatal error handling for these stdio functions, maybe it makes sense to have `safe_fopen`, `safe_fread`, etc. which log and exit?

Reply:
I think `safe_fopen` is good idea because there are several `fopen_or_die` code, but `fread` is lonely here, so `safe_fread` does not help readers of this program.